src/components/dialogs/parameters/sensi/sensi-parameters-selector.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useCallback, useState } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import Grid from '@mui/material/Grid';
+
+import { EQUIPMENT_TYPES } from '../../../utils/equipment-types';
+import { Tab, Tabs } from '@mui/material';
+import { TabPanel } from '../parameters';
+import SensiInjectionsSet from './sensi-injections-set';
+import SensiInjection from './sensi-injections';
+import SensiHVDCs from './sensi-hvdcs';
+import SensiPSTs from './sensi-psts';
+import SensiNodes from './sensi-nodes';
+
+export const MONITORED_BRANCHES_EQUIPMENT_TYPES = [
```

Comment:
I think these declarations have to be on a separate file

Reply:
done